columnflow/ml/__init__.py

```diff
@@ -115,9 +116,12 @@ class MLModel(Derivable):
     # falls back to cls_name if None
     store_name: str | None = None
 
+    # default setting wether to run the preparation_producer in MLEvaluation
```

Comment:
Small typo, `whether`.
Also, maybe this sentence could be

```
# flag denoting whether the preparation_producer is invoked before evaluate()
```

to make it more clear when the producer is called exactly.